src/admin/previews.js

```diff
@@ -10,6 +10,7 @@ import getResourceMetadataLabelFilter from "../_filters/getResourceMetadataLabel
 import site from "../_data/site.json";
 import imagePositionWithTextShortcode from "../_shortcodes/image-position-with-text.js";
 import getId from "../_utils/extract-youtube-id.js";
+import slugify from "../../node_modules/@11ty/eleventy/node_modules/@sindresorhus/slugify";
```

Comment:
I think you should import this package as just `@sindresorhus/slugify`, rather than the one within Eleventy

Reply:
Thanks Ned, we imported slugify this way in another project if I remember, what is the correct syntax for it?

Reply:
Never mind, happened to be that my node_modules was outdated and didn't have @sindresorhus/slugify